end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<p>General feedback: I know the intent of this collection of notebooks is to showcase SageMaker functionalities, but since the target audiences include data scientists, I think it would maximize their user experience by including some interpretation of the visualizations.</p><p> </p><p>Also, it's not clear to me what are you trying predict. I suppose you are trying to predict which claim is a fraud, but it is not clearly stated. And If I were a data scientist, I would be wondering how the vis helped me to understand the data set and the predictive objective. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>done</p>